components/motor/ulnstepper/28byj-48.go

```diff
@@ -0,0 +1,368 @@
+// Package unipolarfivewirestepper implements a GPIO based
+// stepper motor (model: 28byj-48) with uln2003 controler.
+package unipolarfivewirestepper
```

Comment:
this package name makes it seem as though this code is usable for _all_ unipolar five-wire stepper motors. Is this the case? If so, the name of the constructor and the struct in this file should probably change. If not, I think this package name is more appropriate as being the motor name for now?